### ISSUE TITLE
Create form to allow members to opt-in to a member directory

### ISSUE THREAD

**Issue by jonathandavidlewis** [san-diego-tech-hub/sdth-site#164]
Form fields:

Name (required)
email address:
github handle:
linkedIn url:
Areas of interest: SDTH Site, Mentorship, Mentee, Startup, Networking, Job Search, hiring, 
Notes:

Note that users should have a way to opt out or update their information. If the info is in a live google sheet, they can be required to login to fill it out, and allow only 1 response, but allow the response to be updated.

**Comment by jonathandavidlewis** [comment]
@appleJax
What do you think of the fields?

**Comment by jonathandavidlewis** [comment]
Can you link anyone else here?

**Comment by appleJax** [comment]
@jonathandavidlewis Yeah, I like that idea. "Areas of Interest" should be predefined to make things easy to search. We can add more options as we think of them. We can add other social media platforms (twitter, fb, etc), and I think the fields "website" and "company" would be useful as there are a lot of entrepreneurs / company representatives in the community.
